api-samples/python/create_survey.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python2.7
+
+import argparse
+
+from googleapiclient.errors import HttpError
+
+from oauth import get_service_account_auth
+
+
+def create_survey(cs, owner_emails):
```

Comment:
Could you update these references to "cs" (which stood for "consumer surveys") to "surveys" instead?